internal/router/epochs.go

```diff
@@ -0,0 +1,61 @@
+package router
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gofiber/fiber/v2"
+
+	"github.com/spacemeshos/explorer-backend/model"
+)
+
+func (a *AppRouter) epochs(ctx *fiber.Ctx) error {
+	pageNum, pageSize := a.getPagination(ctx)
+	epochs, epochsTotal, err := a.appService.GetEpochs(ctx.UserContext(), pageNum, pageSize)
+	if err != nil {
+		return fmt.Errorf("failed to get epoch list: %w", err)
+	}
+	return ctx.JSON(fiber.Map{"data": epochs, "pagination": a.setPaginationResponse(epochsTotal, pageNum, pageSize)})
+}
+
+func (a *AppRouter) epoch(ctx *fiber.Ctx) error {
+	layerNum, _ := strconv.Atoi(ctx.Params("id"))
+	epochs, err := a.appService.GetEpoch(ctx.UserContext(), layerNum)
+	if err != nil {
+		return fmt.Errorf("failed to get epoch info: %w", err)
+	}
+	return ctx.JSON(fiber.Map{"data": []*model.Epoch{epochs}})
+}
+
+func (a *AppRouter) epochDetails(ctx *fiber.Ctx) error {
+	pageNum, pageSize := a.getPagination(ctx)
+	epochID, err := strconv.Atoi(ctx.Params("id"))
+	if err != nil {
+		ctx.Status(http.StatusBadRequest)
+		return fmt.Errorf("wrong epoch id: %w", err)
+	}
+	var (
+		response interface{}
+		total    int64
+	)
+
+	switch ctx.Params("entity") {
```

Comment:
I can see this switch 4 times in this PR. Does it make sense to introduce string constants "layers", "txs", etc?